### PR TITLE
Remove non-existing affix

### PR DESCRIPTION
**Pull request by avilene**
Tyrannical raging volcanic awakened

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -255,12 +255,6 @@ <h4 class="title title--small trn">Weekly schedule for affixes</h4>
                             	<td class="table__cell"><span class="trn">Tyrannical</span></td>
                                 <td class="table__cell"><span class="trn">Raging</span></td>
                                 <td class="table__cell"><span class="trn">Necrotic</span></td>
-                                <td class="table__cell"><span class="trn">Awakened</span></td>
-							</tr>
```
Comment (domainzero) [inline index.html]: These are from the previous line in the table, so they should stay.

File: index.html [hunk 2]
```diff
@@ -255,12 +255,6 @@ <h4 class="title title--small trn">Weekly schedule for affixes</h4>
...
-                            <tr id="awvoraty" class="table__row">
-                            	<td class="table__cell"><span class="trn">Tyrannical</span></td>
-                                <td class="table__cell"><span class="trn">Raging</span></td>
-                                <td class="table__cell"><span class="trn">Volcanic</span></td>
                                 <td class="table__cell"><span class="trn">Awakened</span></td>
 							</tr>
```
Comment (domainzero) [inline index.html]: These are from the table entry you want to remove, so they should go.

File: index.html (hunk 1 above)
Comment (avilene) [reply index.html]: This is just the UI telling you that its taken away but the net result is exactly the same.

File: index.html (hunk 2 above)
Comment (avilene) [reply index.html]: This is just the UI telling you that its taken away but the net result is exactly the same.

**Comment by avilene** [comment]
I don't see an issue here, i deleted it as a whole chunk but git recognizes it as if the top one should be removed rather than the bottom instance of Awakened.

**Comment by domainzero** [comment]
weird, alright! will merge soon. thanks!

**Pull request merged**